### DISCUSSION TITLE
Restart kernel button disappeared

### DISCUSSION THREAD

**Discussion by oscar-defelice** [microsoft/vscode-jupyter#10758] [Questions and Answers]
In my visual studio code, I do not have the restart and stop execution buttons.

<img width="781" alt="image" src="https://user-images.githubusercontent.com/49638680/178236847-a71d223e-347f-43ac-8867-70a1a759e3dd.png">

If I look at the extension log I find this.

<img width="1075" alt="image" src="https://user-images.githubusercontent.com/49638680/178236323-e0de880c-291c-4afe-a8b9-5e1467e7968c.png">

In particular it says `jupyter.restartkernel` is not defined in section 'commands'. As well as `jupyter.interruptkernel`and `jupyter.interactive.exportasnotebook`.

How can I do to reset such properties in the settings.json file?

**Comment by IanMatthewHuff** [comment]
@oscar-defelice Thanks for reporting this and sorry about the issue. I believe that we have a localization issue hitting some users currently. Looking into it.
